tools/git-version.ts

```diff
@@ -57,8 +57,8 @@ const commitAndTag = async (
     branch
   }: { deliverinoPrivateKey: string; owner: string; repo: string; branch: string }
 ) => {
-  if (branch !== 'main') {
-    throw new Error('Version bumps are only allowed on the main branch!')
+  if (branch !== MAIN_BRANCH) {
```

Comment:
Restricting the job to be done only on main should not only be done here.

Reply:
Why not? This makes very much sense imo. Bumping the version on other branches and not updating the version on main breaks the next delivery. There is also no reason why we would want to deliver from another branch and therefore no reason to bump the version on other branches.

Reply:
We even had a ticket for this which was discussed: https://issues.tuerantuer.org/browse/IGAPP-5

Reply:
Yes it should be restricted to main. But I think the check should be in the bump_version step and not the command here. But it is check in both places so nevermind.